### PR TITLE
1810 Anten Skrabec

### PR DESCRIPTION
**Pull request by cebarks**

### CODE REVIEW

File: test/curator_test.rb [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+  def test_it_starts_with_no_artists
+    assert_equal [], @curator.artists
+  end
+
+  def test_it_starts_with_no_photographs
+    assert_equal [], @curator.photographs
+  end
+
+  def test_it_can_add_photographs
+    setup_photos
```
Comment (BrianZanti) [inline test/curator_test.rb]: I would prefer in the `test_it_can_add_photographs` that you call the `add_photograph` method. In order to do this, you would have to ignore your setup_photos

File: test/curator_test.rb [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+  def test_it_can_find_artists_with_multiple_photographs
+    setup_artists
+    setup_photos
+
+    diane_arbus = @curator.find_artist_by_id("3")
+    actual = @curator.artists_with_multiple_photographs
+
+    assert_equal 1, actual.length
+    assert_equal diane_arbus, actual.first
```
Comment (BrianZanti) [inline test/curator_test.rb]: Instead of these two assertions, you could do `assert_equal [diane_arbus], actual`

File: test/curator_test.rb [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+  end
+
+  def test_photographs_taken_by_artists_from
+    setup_artists
+    setup_photos
+
+    actual = @curator.photographs_taken_by_artists_from("United States")
+    assert_equal 3, actual.length
```
Comment (BrianZanti) [inline test/curator_test.rb]: I would prefer this assertion was a little more explicit.

File: lib/curator.rb [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+      find_photographs_by_artist(artist).length <= 1
+    end
+  end
+
+  def find_photographs_by_artist(artist)
+    @photographs.select { |photo| photo.artist_id == artist.id}
+  end
+
+  def photographs_taken_by_artists_from(country)
+    @photographs.select { |photo| find_artist_by_id(photo.artist_id).country == country}
```
Comment (BrianZanti) [inline lib/curator.rb]: I would not use the single line syntax here. There is a lot going on in this one line.

File: lib/curator.rb [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+  end
+
+  def load_artists(file)
+    FileIO.load_artists(file).each do |args|
```
Comment (BrianZanti) [inline lib/curator.rb]: Great job using the code we gave you

File: test/curator_test.rb [hunk 6]
```diff
@@ -0,0 +1,161 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+
+require './lib/curator'
+
+class CuratorTest < Minitest::Test
```
Comment (BrianZanti) [inline test/curator_test.rb]: I think overall your tests look pretty good!

**Comment by BrianZanti** [comment]
Great job! I think your coding is exceptionally clean.

File: test/curator_test.rb (hunk 3 above)
Comment (cebarks) [reply test/curator_test.rb]: Maybe an array of the artists instead?

**Pull request closed without merging**